src/forms.css

```diff
@@ -42,10 +42,14 @@
 
 .input:focus,
 .textarea:focus,
-.select:focus {
+.select.select--stroke:focus {
```

Comment:
I don't think we need the `.select` part of this selector, can be .select--stroke:focus

Reply:
This one is a bit interesting. When the `.select:focus { box-shadow: none }` rule is included, the `.select` part of the selector is necessary. Otherwise, a select element using a `select--stroke` modifier loses its border when focused. Here's a screenshot where the focus is currently on the first stroked element when `.select` is removed from the above selector. 

<img width="768" alt="Screen Shot 2021-04-14 at 2 24 45 PM" src="https://user-images.githubusercontent.com/15935667/114781303-59c36980-9d2d-11eb-9fca-3fb8b2cf9f96.png">


Would be happy to hear ideas for better approaches! This also might change depending on how we decided to handle the `.select:focus` rule (i.e. the comment below).

Reply:
aha, that makes sense.